src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -705,10 +705,10 @@
       <field ref="remarks" in-xml="WITH_WRAPPER"/>
     </model>
     <constraint>
-      <allowed-values target="prop/@name" allow-other="yes">
-        <enum value="control-origination">Identifies the source of the implemented control.</enum>
+      <allowed-values target="(.|statement|.//by-component)/prop/@name" allow-other="yes">
+        <enum value="control-origination">Identifies the source of the implemented control.  Any <code>control-origination</code> prop defined in a child context will override the parent value.</enum>
       </allowed-values>
-      <allowed-values target="prop[@name='control-origination']/@value">
+      <allowed-values target="(.|statement|.//by-component)/prop[@name='control-origination']/@value">
         <enum value="organization">The control is implemented by the organization owning the system, but is not specific to the system itself.</enum>
```

Comment:
So I tried doing something different and I am looking at [content in ISM ](https://www.cyber.gov.au/ism/) now that they [recently released some things in OSCAL](https://www.cyber.gov.au/ism/oscal). On that note, they have an interesting overlapping set of notions between control origination and responsible parties for a control that seems relevant to us in this PR. The only thing I don't see how we cover with your wonderful additions today, but you could maybe bend the truth on, is what they refer to as "Outsourced Service Provider" which, as you can imagine could be `inherited` or `customer-configured` based on your interpretation. The current OSCAL catalog for ISM won't help but [they have this handy spreadsheet with an Info sheet including guidance](https://www.cyber.gov.au/sites/default/files/2022-09/Cloud%20Controls%20Matrix%20Template%20%28September%202022%29.xlsx). But maybe I am off here? A screenshot of the relevant section.

![image](https://user-images.githubusercontent.com/94922603/194122127-289fe53c-300c-46ef-9d7b-824bed135090.png)



Would love to meet and discuss, with you Chris, maybe include Dave (depending on timeline, but I can tell the PR isn't going anywhere because of our CODEOWNERS config, lol)/

Reply:
@aj-stein-nist Can we put this in a new issue?  Since the current PR was related to the property, and the schema would not block someone from adding custom values (not limited the the allow-values), this might be something we should fully evaluate, but at the same time not hold the current change back.

Reply:
And done.

https://github.com/usnistgov/OSCAL/issues/1502